pkg/config/config.go

```diff
@@ -838,10 +836,10 @@ var (
 // CONTAINERS_CONF environment variable.  This is usually done for testing.
 func Default() (*Config, error) {
 	var err error
-	configOnce.Do(func() {
-		config, err = NewConfig("")
-	})
-	return config, err
+	if defaultConf == nil {
```

Comment:
We're hitting that in the 1.14 branch as well (see https://github.com/containers/common/pull/189). I'd rather remove the sync.Once entirely.

If there's a race, the other thread will wait for. But `NewConfig("")` is also reading system paths which we do not consider, so an unconditional singleton seems wrong to me.

I'd just remove the singleton.